src/client/kinto_http.rs

```diff
@@ -90,6 +103,11 @@ pub fn get_changeset(
     cid: &str,
     expected: u64,
 ) -> Result<ChangesetResponse, KintoError> {
+    debug!(
+        "The expected timestamp for bucket={}, collection={} is {}",
```

Comment:
in most places (and other clients), we refer as a collection like this : `{bid}/{cid}`

Reply:
done